tests/utils/test_tests.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from airflow.utils.tests import assertEqualIgnoreMultipleSpaces
+from contextlib import contextmanager
+
+
+class UtilsTestsTest(unittest.TestCase):
+
+    @contextmanager
+    def assertNotRaises(self, error_type):
+        try:
+            yield None
+        except error_type:
+            raise self.failureException('{} raised'.format(error_type.__name__))
+
+    def test_assertEqualIgnoreMultipleSpaces_raises(self):
+        str1 = 'w oo f'
+        str2 = 'meow'
+
+        self.assertRaises(AssertionError, lambda: assertEqualIgnoreMultipleSpaces(self, str1, str2))
+
+    def test_assertEqualIgnoreMultipleSpaces_passes(self):
+        str1 = 'w oo f'
+        str2 = """
+            w
+            oo    f
+        """
+
+        with self.assertNotRaises(AssertionError):
+            assertEqualIgnoreMultipleSpaces(self, str1, str2)
```

Comment:
Why can't we simply have

```python
assertEqualIgnoreMultipleSpaces(self, str1, str2)
```
instead of 
```python
with self.assertNotRaises(AssertionError):
    assertEqualIgnoreMultipleSpaces(self, str1, str2)
```
 It can capture properly as well.

Please let me know if I missed anything. Thanks.

Reply:
@XD-DENG Removed unnecessary lines 🙂

Reply:
Will merge once CI passes. Thanks!